bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/AbstractHomeConnectThingHandler.java

```diff
@@ -1006,27 +1008,39 @@ protected ChannelUpdateHandler defaultAmbientLightChannelUpdateHandler() {
                     boolean enabled = data.getValueAsBoolean();
                     if (enabled) {
                         // brightness
-                        Data brightnessData = apiClient.get().getAmbientLightBrightnessState(getThingHaId());
-                        getThingChannel(CHANNEL_AMBIENT_LIGHT_BRIGHTNESS_STATE)
-                                .ifPresent(channel -> updateState(channel.getUID(),
-                                        new PercentType(brightnessData.getValueAsInt())));
+                        getThingChannel(CHANNEL_AMBIENT_LIGHT_BRIGHTNESS_STATE).ifPresent(channel -> {
+                            try {
+                                Data brightnessData = apiClient.get().getAmbientLightBrightnessState(getThingHaId());
+                                updateState(channel.getUID(), new PercentType(brightnessData.getValueAsInt()));
+                            } catch (CommunicationException | ApplianceOfflineException | AuthorizationException e) {
+                                updateState(channel.getUID(), UnDefType.UNDEF);
```

Comment:
Shouldn't you set the Thing status to OFFLINE here instead?

Reply:
I am tempted to undo my change for this case.
By the way, catching the 3 kinds of exceptions was a bad idea.

Reply:
Undone this change